### DISCUSSION TITLE
Modelling DSL Interface

### DISCUSSION THREAD

**Discussion by JokerCSL** [netbox-community/netbox#7522] [General]
Hi,

I'm trying to model a DSLAM and don't know what Interface Type to choose for DSL.
Could I somehow create my own Interface Type?

Any hints much appreciated.

Best Regards

**Comment by jeremystretch** [comment]
An xDSL interface type was added in v3.0.3 (see #6387).

**Reply by candlerb** [reply]
Just to add: you can't create your own interface types, but there is type "Other" which can be used when there's no exact match to your requirements.  If you think your interface type is generally useful, you can raise a feature request via github to have it added.

**Comment by JokerCSL** [comment]
Thanks a lot for your answers.
Seems we need to upgrade to 3.x asap - still waiting for Credential Management (Secrets) to be available there.

**Reply by julianze** [reply]
https://github.com/DanSheps/netbox-secretstore

**Comment by JokerCSL** [comment]
Is this the official continuation of the Secrets functionality in Netbox?

**Reply by julianze** [reply]
Yes, look at the readme: 
`This is the continuation of the secrets app.`
